### DISCUSSION TITLE
Is there any way to save the relation id field directly?

### DISCUSSION THREAD

**Discussion by gfpacheco** [prisma/prisma#2097] [Q&A] [answered]
I read all about [relation writes](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/relation-queries#nested-writes).

But I'm writing a code generation tool, and it would be much easier if I could just write to the relation foreign key field directly.

Imagine I have this schema:

```
model User {
  id    Int    @default(autoincrement()) @id
  posts Post[]
}

model Post {
  id       Int    @default(autoincrement()) @id
  author   User   @relation(fields: [authorId], references: [id])
  authorId Int
}
```

Is there any way to create a post that looks something like this:

```
prisma.posts.create({ data: { authorId: 1 } });
```

Instead of:

```
prisma.posts.create({ data: { author: { connect: { id: 1 } } } });
```

*Update*: Feature request [created](https://github.com/prisma/prisma/issues/2111)

**Comment by nikolasburk** [comment, marked as answer]
Hey @gfpacheco, this is currently not supported unfortunately but I think this is already planned! Do you mind creating a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) to ask that the relation scalar field should be writable in the Prisma Client API? 🙏

**Reply by u007** [reply]
is this still pending?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
